packages/splits-kit/storybook/constants/chains.ts

```diff
@@ -141,4 +142,12 @@ export const STORYBOOK_CHAIN_INFO: ChainInfo = {
       `https://base-mainnet.g.alchemy.com/v2/${process.env.STORYBOOK_ALCHEMY_API_KEY}`,
     ],
   },
+  [shape.id]: {
+    ...CHAIN_INFO[shape.id],
+    viemChain: shape,
+    rpcUrls: [
+      // TODO: rpc url
```

Comment:
lol, why do we have this on every single one...

Reply:
lol